### PR TITLE
adding refreshItemThumbnail function

### PR DESCRIPTION
**Pull request by mhines-usgs**
@jiwalker-usgs can you review this?

### CODE REVIEW

**Comment by jread-usgs** [comment]
@mhines-usgs did you forget to add the .R file?

**Comment by mhines-usgs** [comment]
@jread-usgs *blush*

**Comment by jread-usgs** [comment]
@mhines-usgs testbot wants you to add a test.

File: R/setBaseURL.R [hunk 1]
```diff
@@ -41,4 +41,5 @@ setBaseURL = function(endpoint="prod"){
   pkg.env$auth_check = paste0(pkg.env$url_base, "security/auth/check/")
   pkg.env$authToken = NULL
   pkg.env$username = NULL
+  pkg.env$item_thumbnail = paste0(pkg.env$url_base, "data/thumbnail/item/")
```
Comment (jiwalker-usgs) [inline R/setBaseURL.R]: I'd move this up a few lines with the rest of the item urls.

File: R/refreshItemThumbnail.R [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#'@title Refreshes an item's thumbnail PNG 
+#'@description returns a boolean success 
+#'@param itemID id of item to refresh thumbnail for
+#'@param ... extra parameters to pass to checkAuth
+#'@return returns TRUE if the thumbnail is put successfully, and false if it fails
+#'#'@importFrom base64enc base64encode
```
Comment (jiwalker-usgs) [inline R/refreshItemThumbnail.R]: double comment

**Comment by jread-usgs** [comment]
@mhines-usgs can you also check that the item exists before doing the `PUT`? Fail if it doesn't? Otherwise a typo will be bad bear.

**Comment by mhines-usgs** [comment]
just committed a check item change to the refreshItemThumbnail

**Pull request merged**